pom.xml

```diff
@@ -75,14 +75,14 @@
         -->
 
         <!-- versions of "subprojects" of Red Hat Fuse -->
-        <version.io.hawt>2.0.0.fuse-730043</version.io.hawt>
+        <version.io.hawt>2.0.0.fuse-730074</version.io.hawt>
```

Comment:
@cunningt @orpiske @oscerd 
hmm, if the build is for 7.4, why we have prefixes 730? I think I changed that... https://github.com/jboss-fuse/fuse-jenkins-pipelines/blob/master/fuse/7.4/Jenkinsfile#L72

Maybe PR generator job has to be adjusted as well...

Reply:
Yeah, that looks wrong to me as well.